readme.md

```diff
@@ -54,6 +54,24 @@ import ky from 'ky';
 })();
 ```
 
+<details>
+
+<summary>Also you can use it in es5 environment. Click here to see how</summary>
+
```

Comment:
This also needs to mention the requirement of the fetch polyfill.

Reply:
And a dependency on `@babel/polyfill` as well (or specifically `core-js@2` and `regenerator-runtime`).